ws-service/server.ts

```diff
@@ -44,25 +44,55 @@ const addressRouteConnection = (socket: Socket): void => {
   console.log('  total:', countA)
 }
 
+const resolveOpReturn = (opr: any) => {
+  try {
+    if (!(typeof opr === 'string' && opr !== '')) {
+      throw new Error("Error parsing Opreturn options")
```

Comment:
We actually shouldn't error if this is `''` , we should just return `null`. `Transaction`s with `opReturn` column equal to `''` are just txs with specific data that is not relevant to us OR empty. This will be any tx that an address has received that originated NOT from a paybutton (client) or from a paybutton created before than last month (when we implemented this OP_RETURN data), so this will be majority of txs.

BTW, `''` is the NULL in the DB for string values, as per the convention we use in this repo, since having string values actually be nullable in the DB results in two possible null values:  `''` and `NULL`, so there are no string values nullable in the DB, to avoid ambiguity.

Reply:
also, `!(typeof opr === 'string' && opr !== '')` is a more deep way of saying `(typeof opr !== 'string' || opr === '')`, which is simpler and logical equivalent

Reply:
the logic in the IF is wrong indeed, thanks for the review, I will refactor this.